etc/kayobe/kolla.yml

```diff
@@ -457,7 +457,7 @@ kolla_build_args:
 #kolla_enable_ceilometer_horizon_policy_file:
 #kolla_enable_ceilometer_ipmi:
 #kolla_enable_cells:
-kolla_enable_central_logging: true
```

Comment:
Please don't modify these global flags.

Reply:
This has been reverted and the disabling of services has been moved into `ci-multinode/kolla.yml`